datalad/interface/tests/test_add_archive_content.py

```diff
@@ -272,15 +272,15 @@ def d2_basic_checks():
     ok_archives_caches(repo.path, 0, persistent=False)
 
     repo.drop(opj('1', '1 f.txt'))  # should be all kosher
-    repo.drop(key_1tar, options=['--key'])  # is available from the URL -- should be kosher
+    repo.drop(key_1tar, key=False)  # is available from the URL -- should be kosher
```

Comment:
reviewing your own code is always so useful -- not sure how it worked out here ;-)